R/PackageInheritanceReporter.R

```diff
@@ -1,6 +1,7 @@
-#' @title Package Class Inheritance Reporter Class
+#' @title Class Inheritance Reporter
 #' @name InheritanceReporter
-#' @family PackageReporters
+#' @family Network Reporters
```

Comment:
@jayqi what's the logic behind having `Network Reporter` and `Package Reporter` being a distinction in the user docs?  It causes them to show up twice in the "See Also" section of the docs.

Reply:
I'm open to changing this. `Package Reporters` are a superset of `Network Reporters`. Having both means that you'll see the other reporters if you're looking at the roxygen article for `SummaryReporter`. 

We could manually generate the @seealso section as well to not have the repeats.

Reply:
All in all, two links at the bottom of a doc is no big deal.